### PR TITLE
Feature/random split

### PR DESCRIPTION
**Pull request by michcio1234**
Required to implement `from_delayed`.

### CODE REVIEW

File: sparsity/dask/core.py [hunk 1]
```diff
@@ -59,6 +59,36 @@ def finalize(results):
     return sp.SparseFrame.vstack(results)
 
 
+def from_delayed(dfs, meta=None, divisions=None, prefix="from-delayed"):
+    for df in dfs:
+        if not isinstance(df, Delayed):
+            raise TypeError("Expected Delayed object, got %s" % type(df).__name__)
+
+    if meta is None:
+        meta = delayed(make_meta)(dfs[0]).compute()
```
Comment (kayibal) [inline sparsity/dask/core.py]: I'd raise a warning here, this computation might be very expensive...

File: sparsity/dask/core.py [hunk 2]
```diff
@@ -59,6 +59,36 @@ def finalize(results):
...
+    else:
+        meta = make_meta(meta)
+
+    name = prefix + "-" + tokenize(*dfs)
+    dsk = merge(df.dask for df in dfs)
```
Comment (kayibal) [inline sparsity/dask/core.py]: Sure you can use `dask` attribute here? I'm not sure if that is not deprecated by dask collection and the latest API suggests to use `.__dask_graph__()`

File: sparsity/test/test_dask_sparse_frame.py [hunk 3]
```diff
@@ -541,3 +589,26 @@ def test_persist(dsf):
     res = persisted.compute().todense()
 
     pdt.assert_frame_equal(res, correct)
+
+
+@pytest.fixture(scope='session')
+def dsf_arange(sf_arange):
```
Comment (kayibal) [inline sparsity/test/test_dask_sparse_frame.py]: I would prefer to have fixtures at the top

File: sparsity/dask/core.py (hunk 1 above)
Comment (michcio1234) [reply sparsity/dask/core.py]: It's exactly like this in dask.

File: sparsity/dask/core.py (hunk 2 above)
Comment (michcio1234) [reply sparsity/dask/core.py]: Dask is using it internally all the time. This is just copied.

Comment (michcio1234) [reply sparsity/dask/core.py]: I also didn't find information that it's deprecated.

**Pull request merged**